Tests/HDWalletTests.swift

```diff
@@ -33,7 +33,8 @@ class HDWalletTests: XCTestCase {
     func testDeriveBitcoin() {
         let wallet = HDWallet(mnemonic: words, passphrase: passphrase)
         let key = wallet.getKey(at: Bitcoin().derivationPath(at: 0))
-        let publicKey = key.publicKey(compressed: true)
+        let publicKey = key.publicKey().compressed
+        XCTAssertEqual(publicKey.description, key.publicKey(compressed: true).description)
 
         XCTAssertEqual("026fc80db3a34e7c6bfc6d7d9a53aeba9e706b309c9cf7ee96fa9c36ff7fd92a20", publicKey.description)
         XCTAssertEqual("3FJjnZNXC6FWQ2UJAaKL3Vme2EJavfgnXe", publicKey.bitcoinAddress(prefix: Bitcoin().payToScriptHashAddressPrefix).description)
```

Comment:
`publicKey.bitcoinAddress(prefix: Bitcoin().payToScriptHashAddressPrefix)`
=>
`key.publicKey().bitcoinAddress(prefix: Bitcoin().payToScriptHashAddressPrefix)`

Can we do it this way? Our goal to make sure it will be using compressed version without specifying it previously